Superkaramba doesn't compile with LDFLAGS="-Wl,--as-needed", this patch fixes that
Patch by Steffen Hau <steffen@hauihau.de>

--- superkaramba-4.0.1/superkaramba/src/CMakeLists.txt	2008-02-07 18:45:04.000000000 +0100
+++ superkaramba-4.0.1.new/superkaramba/src/CMakeLists.txt	2008-02-07 18:46:52.000000000 +0100
@@ -82,6 +82,16 @@
 KDE4_ADD_LIBRARY(superkaramba SHARED ${superkaramba_LIB_SRCS})
 set_target_properties(superkaramba PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
 
+FIND_LIBRARY(LIBDL dl)
+IF(LIBDL)
+  LIST(APPEND PYTHON_LIBRARIES ${LIBDL})
+ENDIF(LIBDL)
+
+FIND_LIBRARY(LIBUTIL util)
+IF(LIBUTIL)
+  LIST(APPEND PYTHON_LIBRARIES ${LIBUTIL})
+ENDIF(LIBUTIL)
+
 TARGET_LINK_LIBRARIES(superkaramba
 ${KDE4_KIO_LIBS}
 ${KDE4_KPTY_LIBS}
